spec/system/referee_interface/referee_can_use_initial_and_chaser_sign_in_links_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'Referee can use sign in link in the initial and chaser email', sidekiq: true do
+  scenario 'Referee clicks sign in links on the initial and chaser reference request emails' do
+    given_the_confirm_relationship_and_safeguarding_feature_flag_is_active
+    and_i_am_a_referee_of_an_submitted_application
+    and_i_received_the_initial_reference_request_email
+    when_i_click_on_the_link_within_the_email
+    then_i_am_asked_to_confirm_my_relationship_with_the_candidate
+
+    given_i_received_the_chaser_reference_request_email
+    when_i_click_on_the_link_within_the_email
```

Comment:
Should we also test if the link in the original email still works?

Reply:
What do you mean by "original" email?

Reply:
The first non-chaser one* 😅

Reply:
Ah yes, we do. See:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/3437ead947f9fe6c890cd69aa8a587c689539e3a/spec/system/referee_interface/referee_can_use_initial_and_chaser_sign_in_links_spec.rb#L7-L9

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/3437ead947f9fe6c890cd69aa8a587c689539e3a/spec/system/referee_interface/referee_can_use_initial_and_chaser_sign_in_links_spec.rb#L25-L27

Reply:
Resolved in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1644.